src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/HttpBasicLdapFactory.php

```diff
@@ -45,7 +45,11 @@ public function create(ContainerBuilder $container, string $id, array $config, s
         ;
 
         // entry point
-        $entryPointId = $this->registerEntryPoint($container, $id, $config, $defaultEntryPoint);
+        $entryPointId = 'security.authentication.basic_entry_point.'.$id;
```

Comment:
In `HttpBasicFactory` the code is different:
```
// entry point
$entryPointId = $defaultEntryPoint;
if (null === $entryPointId) {
  $entryPointId = 'security.authentication.basic_entry_point.'.$id;
  $container
    ->setDefinition($entryPointId, new ChildDefinition('security.authentication.basic_entry_point'))
    ->addArgument($config['realm'])
  ;
}
```

Reply:
good catch, fixed